miniapps/nurbs/CMakeLists.txt

```diff
@@ -50,6 +50,11 @@ add_mfem_miniapp(nurbs_ex5
   LIBRARIES mfem)
   add_dependencies(nurbs_ex5 copy_miniapps_nurbs_data)
 
+add_mfem_miniapp(nurbs_ex10
+  MAIN nurbs_ex10.cpp
```

Comment:
Also add to `makefile` (this example is not built with `make`). Similarly for `nurbs_ex10p.cpp`.

Reply:
Done. Also updated .gitignore